modules/platforms/cpp/odbc/include/ignite/odbc/connection.h

```diff
@@ -204,25 +204,27 @@ namespace ignite
              *
              * @param req Request message.
              * @param rsp Response message.
-             * @param timeout Timeout.
+             * @param timeout Timeout. 0 means disabled and -1 means to use default connection timeout.
```

Comment:
```suggestion
             * @param timeout0 Timeout. 0 means disabled and -1 means to use default connection timeout.
```
On the other hand, `timeout0` as a parameter name looks weird, maybe we should use different variable names?

Reply:
+1, why not just
```
int32_t actualTimeout = timeout < 0 ? this->timeout : timeout
```

Reply:
I am for renaming private field to `timeout_`, the current code style really sucks and error prone

Reply:
Agree